test/runtime/Policies.go

```diff
@@ -56,7 +57,7 @@ var _ = Describe("RuntimePolicyEnforcement", func() {
 		if initialized == true {
 			return
 		}
-		logger = log.WithFields(log.Fields{"testName": "RuntimePolicyEnforcement"})
+		logger = logrus.WithFields(logrus.Fields{"testName": "RuntimePolicyEnforcement"})
```

Comment:
Oops! this was definitely a mistake. You still want to use the package local `log` variable (in log.go) but `logrus.Fields` is the correct thing, and you'd need to import the package as you have.